Interface/js/fetch.js

```diff
@@ -0,0 +1,35 @@
+async function fetchData(url, method, jwtToken, bodyData) {
```

Comment:
Did your own vanilla fetch wrapper??

Reply:
Yes, it's just to make the request pipeline easier to understand